locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -420,127 +428,132 @@ msgstr "Modifier « %(book_title)s »"
 msgid "Add Book"
 msgstr "Ajouter un livre"
 
-#: bookwyrm/templates/book/edit_book.html:40
+#: bookwyrm/templates/book/edit_book.html:54
 msgid "Confirm Book Info"
 msgstr "Confirmer les informations de ce livre"
 
-#: bookwyrm/templates/book/edit_book.html:47
+#: bookwyrm/templates/book/edit_book.html:62
 #, python-format
 msgid "Is \"%(name)s\" an existing author?"
-msgstr "Est‑ce que l’auteur ou l’autrice « %(name)s » existe déjà ?"
+msgstr "Est‑ce que l’auteur/autrice « %(name)s » existe déjà ?"
 
-#: bookwyrm/templates/book/edit_book.html:52
+#: bookwyrm/templates/book/edit_book.html:71
 #, python-format
 msgid "Author of <em>%(book_title)s</em>"
-msgstr "Commencer « <em>%(book_title)s</em> »"
+msgstr "Auteur/autrice de <em>%(book_title)s</em>"
 
-#: bookwyrm/templates/book/edit_book.html:55
+#: bookwyrm/templates/book/edit_book.html:75
 msgid "This is a new author"
 msgstr "Il s’agit d’un nouvel auteur ou d’une nouvelle autrice."
 
-#: bookwyrm/templates/book/edit_book.html:61
+#: bookwyrm/templates/book/edit_book.html:82
 #, python-format
 msgid "Creating a new author: %(name)s"
-msgstr "Création d’un nouvel auteur ou d’une nouvelle autrice : %(name)s"
+msgstr "Création d’un nouvel auteur/autrice : %(name)s"
```

Comment:
Du coup, « un nouvel autrice » ne fonctionne pas. :/

Reply:
You gotta love verbosity… The interface in French has 2x the words of the English interface when we're duplicating all the adjectives. The meaning here is quite clear IMO, it's easily understood that "nouvel" also applies to "autrice".

Reply:
I was not implying my translation was perfect as I of course find it too verbose. I was more referring to the fact that we could go without “/autrice” or go with “d’une nouvelle autrice” if we skim for less verbosity without caring for gender concordance.

How to respect both genders equally? Or all genders?